<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUser</class>
 <widget class="QWidget" name="AddUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>652</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1110</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>1080</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1080</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_create">
        <property name="minimumSize">
         <size>
          <width>186</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#btn_create{
background-image: url(:/36.png);
border:none;
}
QPushButton#btn_create:pressed{

background-image: url(:/37.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_back">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#btn_back{
background-image: url(:/按键_上一页2.png);
border:none;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>1080</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="le_passwd">
        <property name="minimumSize">
         <size>
          <width>149</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>149</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,0,0);
font: 75 12pt &quot;Microsoft JhengHei UI&quot;;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="3">
       <widget class="QLabel" name="label_tubiao">
        <property name="minimumSize">
         <size>
          <width>94</width>
          <height>94</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>94</width>
          <height>94</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/图标_普通用户.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="le_passwd2">
        <property name="minimumSize">
         <size>
          <width>149</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>149</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,0,0);
font: 75 12pt &quot;Microsoft JhengHei UI&quot;;
</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_passwd2">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 25px &quot;Microsoft YaHei UI&quot;;
font-size: 25px;
</string>
        </property>
        <property name="text">
         <string>确认六位密码</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_passwd">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 25px &quot;Microsoft YaHei UI&quot;;
font-size: 25px;
</string>
        </property>
        <property name="text">
         <string>设置六位密码</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_name">
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 25px &quot;Microsoft YaHei UI&quot;;

</string>
        </property>
        <property name="text">
         <string>输入用户名</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="le_name">
        <property name="minimumSize">
         <size>
          <width>149</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>149</width>
          <height>31</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0,0,0);
font: 75 12pt &quot;Microsoft JhengHei UI&quot;;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
